server/utils/helpers.js

```diff
@@ -0,0 +1,22 @@
+const { Demand } = require('../models/Demand');
+
+const saveDemand = (companyName) => {
+  Demand.findOne({ param: companyName }, (err, company) => {
+    if (company != null) {
+      company.counter += 1; // eslint-disable-line no-param-reassign
+      company.save();
+    } else {
+      const newDemand = new Demand({ param: companyName });
+      newDemand.save();
+    }
+  });
+};
+
+const getRandomElement = array => (
```

Comment:
Probably this method would be good to have in the global scope of the project, not only as a backend helper... 🤔